metricbeat/docs/fields.asciidoc

```diff
@@ -6040,6 +6040,194 @@ Used inodes
 
 
 [float]
+== events Fields
```

Comment:
The metricset should probably be called `event` and not `events` to be in line with our naming convention.